back-office/wishlist.md

```diff
@@ -0,0 +1,346 @@
+
+I can view the products and I can sort them as follows:  
+- Last added (default sorting)  
+- Price highest to lowest
+- Price lowest to highest 
+
+In all three cases, keep the discount price in mind  
+![wishlist screen 9](https://github.com/PrestaShop/prestashop-specs/blob/master/img/wishlist-screen-9.png)
+
+Please note that if a product is deleted or deactivated from the catalog, the product automatically disappears from the list.  
+
+For each product, you have to go back to the information listed in US n°1.
+As a logged-in user, I can see how many unique favorite products are in a list.  
+
+What type of users can add products to their favorites?
+Only logged-in users, since the Favorites list is displayed in My Account.  
+
+When I access the page with the my favorites lists, I can see the number of products in each of the lists.  
+
+### US 3: Edit and/or delete a favorite product
+##### As a logged-in user, I can edit and/or delete a favorite product  
+Different cases:
+- Products in stock
+- Products out of stock
+- from the product sheet
+- from a favorites list
+
+Functional rules:
+I can modify the quantity, the combinations and the customization of all my favorite products (in stock, out of stock) by clicking on the Pencil in my favorite products list.
+
+When I click on the Pencil, I am redirected to the product sheet to make my modifications. Once the changes are made, you will have to click on the heart to add the new product as a favorite. At this point, the new quantity, combination and / or customization will overwrite the original product in the favorites list.
+
+![wishlist screen 10](https://github.com/PrestaShop/prestashop-specs/blob/master/img/wishlist-screen-10.png)
+
+I can permanently remove a favorite product from the list by clicking on remove and a pop-up appears confirming that the product has been removed from the list.  
+
+You can remove a product from the favorites directly within the product sheet or a product listing, by clicking on the heart (which becomes empty).  
+
+Products with several options:
+- if I delete my product from the product sheet: the product with combinations X and quantity Y will be deleted
+- if I delete my product from a product listing (where I don't have any combination options), it deletes only the product with the default combination if it was added. Otherwise, if I have added a combination as a favorite that isn't the default one, then the product block in the listing will not have the heart filled. The heart is filled only if the merchant has added the product with its default combination. 
+
+Exemple : if I didn't added the product with the default combination in my favorite list
+For example I have a tshirt with following combinations:
+* Red (default combination)
+* Black
+* White
+I add the black and the white to one of my favorite list. It will be impossible to remove these products of the wishlist from the product list because the heart is empty
```

Comment:
```suggestion
Example: I did not add the product with the default combination to my favorites list

For instance, I have a tee-shirt in the following combinations:

* Red (default combination)
* Black
* White

I add the _black_ and the _white_ combinations to my favorites. Both products cannot be removed from the listing page since the heart icon is empty.
```